data/locale/hi/faq.json

```diff
@@ -11,5 +11,8 @@
   "mentor-obligations.md": "GSoC के दौरान एक उपदेशक के रूप में मेरे दायित्व क्या हैं?",
   "thesis.md": "मैं coala के साथ थीसिस लिखना चाहता हूं!",
   "application-template.md": "क्या आपके पास कोई एप्लिकेशन टेम्प्लेट है?",
-  "project-participation.md": "क्या सभी सूचीबद्ध परियोजनाएं GSoC (या कोई अन्य पहल) का हिस्सा होगी?"
-}
+  "project-participation.md": "क्या सभी सूचीबद्ध परियोजनाएं GSoC (या कोई अन्य पहल) का हिस्सा होगी?",
+  "cep-and-its-importance.md": "CEP क्या है? पूर्व-GSoC विश्लेषण करना और प्रस्ताव को जल्द से जल्द डिजाइन करना इतना महत्वपूर्ण क्यों है?",
+  "coala-will-apply-for-gsoc.md": "क्या coala अगले साल गूगल समर ऑफ कोड के लिए आवेदन करेगा?",
+  "high-involvement-status.md": "किसी भी परियोजना की उच्च भागीदारी की स्थिति का क्या अर्थ है?"
+}
```

Comment:
End a file with a newline. Please add it, in this file and in other changed files too.

Reply:
Sure. I'll do the same by evening.